src/Components/Ignitor/src/Ignitor.csproj

```diff
@@ -1,10 +1,10 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <!-- Intentionally targeting netcoreapp3.0 because we do not want to update the Benchmark client to use 3.1 -->
```

Comment:
Please remove this now-invalid comment. @sebastienros will the TFM change break the benchmark client❔